Documentation/Commands/Spaced.md

```diff
@@ -0,0 +1,133 @@
+
+## <img height = '24' src = '../../freecad/ShapeStrings/Resources/Icons/Spaced.svg' /> Spaced ShapeString
+
+Lets you place multiple strings side by side to each other.
+
+The offset you choose always applies, either setting where  
+each string starts relative to the last, or keeping the visible  
+gap between strings constant.
+
+The result is a regular FreeCAD shape object that works with  
+**Part** and **PartDesign** operations for engraving or embossing.
+
+<br/>
+
+## Preview
+
+<img src = '../../Resources/Media/Examples/Spaced.webp' />
+
+![SpacedShapeString icon](./freecad/ShapeStrings/Resources/Icons/Spaced.svg)
```

Comment:
This does not render correctly. Maybe you meant to remove this like Radial?

Reply:
Forgot to remove that.